open-company/open-company-web
# Fix security issue

Update sentry/electron to version 1.3.0 to fix this issue https://github.com/open-company/open-company-web/network/alerts

To test:
- set a Sentry DSN for local dev in `build.boot` (line 293. check instructions at the bottom)
- add the following line at `resources/electron/renderer.js:8`: `window.setTimeout(thisFnDoesntExist, 10000);`
- start the desktop app
- check Sentry has received the error and details are ok

NB: you can use my Sentry electron project
`oc.electron.main/sentry-dsn "https://ecf4ef8092ae4638bf56c0cfa6d50bb0@sentry.io/4944452"`
